src/lib/metricsMgr/MetricsManager.h

```diff
@@ -25,9 +25,62 @@
 *
 * Author: Fermín Galán
 */
-
+#include <sys/time.h>
 #include <string>
 #include <map>
+#include <semaphore.h>
+
+
+
+/* ****************************************************************************
+*
+* Metrics - 
+*/
+#define METRIC_TRANS_IN                            "incomingTransactions"
+#define METRIC_TRANS_IN_REQ_SIZE                   "incomingTransactionRequestSize"
+#define METRIC_TRANS_IN_RESP_SIZE                  "incomingTransactionResponseSize"
+#define METRIC_TRANS_IN_ERRORS                     "incomingTransactionErrors"
+#define METRIC_SERVICE_TIME                        "serviceTime"
+
+#define METRIC_TRANS_OUT                           "outgoingTransactions"
+#define METRIC_TRANS_OUT_REQ_SIZE                  "outgoingTransactionRequestSize"
+#define METRIC_TRANS_OUT_RESP_SIZE                 "outgoingTransactionResponseSize"
+#define METRIC_TRANS_OUT_ERRORS                    "outgoingTransactionErrors"
+
+#define METRIC_TRANSACTIONS                        "transactions"
```

Comment:
Metrics not included in https://github.com/telefonicaid/fiware-orion/issues/2750 are yet unders discussion. If you want to keep them in the .h, I'd suggest to do it in an `#if 0` block.